mods/common/chrome/connection.yaml

```diff
@@ -18,22 +18,21 @@ Background@CONNECTIONFAILED_PANEL:
 			Y: 46
 			Width: PARENT_RIGHT
 			Height: 25
-			Text: Could not connect to AAA.BBB.CCC.DDD:EEEE
```

Comment:
removed in the second commit so automatic extraction didn't detect this

Reply:
This is a misunderstanding on your side. It is already translated. https://github.com/OpenRA/OpenRA/blob/939f715e3c1cd82a080a26d57c29b5f049f6f4bb/OpenRA.Mods.Common/Widgets/Logic/ConnectionLogic.cs#L69